### ISSUE TITLE
"t is not a function" shortly before login-redirection

### ISSUE THREAD

**Issue by thomasbrueggemann** [DevExpress/testcafe#1959]
### Are you requesting a feature or reporting a bug?

Thanks for the effort, @AlexanderMoskovkin .
I used the custom build you provided in https://github.com/DevExpress/testcafe/issues/1938#issuecomment-345649192 and it resolved the previous error regarding "testCafeCore" but it prompts me another error now:

![screenshot 2017-11-23 16 03 51](https://user-images.githubusercontent.com/2313087/33180200-35439db4-d06c-11e7-87bd-1bc698239ee2.png)

### What is the current behavior?

The error appears in the second that TestCafe clicks on the login button and disappears in a blink of an eye when the reload to the logged-in-page happens. But nevertheless shows the test as failed in the terminal-console.

### What is the expected behavior?

I would expect a redirect after testCafe clicks the login button without an error message and without the testcase failing.

### How would you reproduce the current behavior (if this is a bug)?

I used the following test fixture

```js
test("Open showMap.aspx", async t => {
	await t
		.typeText($("#txtUsername-inputEl"), "****")
		.typeText($("#txtPassword-inputEl"), "****")
		.click($("#btnLogin"))
		.expect($("#Header2").exists)
		.ok();
});
```

### Specify your

* operating system: macOS High Sierra 10.13.1
* testcafe version: testcafe-0.18.5-custom20171120
* node.js version: 7.10.1

**Comment by miherlosev** [comment]
Hi, @thomasbrueggemann 
I've reproduced the problem and created a separate issue in our web proxy repository - https://github.com/DevExpress/testcafe-hammerhead/issues/1419

**Comment by LavrovArtem** [comment]
The fix is published in `testcafe@0.18.7-dev20180124`.
